fvm/bootstrap.go

```diff
@@ -637,6 +701,32 @@ transaction(amount: UFix64, recipient: Address) {
 }
 `
 
+const deployLockedTokensTemplate = `
+transaction(publicKeys: [[UInt8]]) {
+    
+    prepare(admin: AuthAccount) {
+        admin.contracts.add(name: "LockedTokens", code: "%s".decodeHex(), admin)
+
+        for key in publicKeys {
+            admin.addPublicKey(key)
+        }
```

Comment:
Now that this is deployed to the service account, we're just re-adding the key that's [already there](https://github.com/onflow/flow-go/pull/997/files#diff-f414ad868524efa32a2d6a821877f9d01b7948e26bece32df25664a8eb4709d8R560). I think we can remove this loop.
```suggestion
```